libs/ui/src/hooks/use_devices.test.ts

```diff
@@ -120,13 +120,13 @@ test('can connect printer as expected', async () => {
   );
 
   // Prevent `act` warning.
-  await waitForNextUpdate();
```

Comment:
Is there still an act warning to prevent here?

Reply:
Yup :/